docker/quickstart/docker-compose.yml

```diff
@@ -15,19 +15,22 @@ services:
       - "3306:3306"
     volumes:
       - ../mysql/init.sql:/docker-entrypoint-initdb.d/init.sql
+      - ${DATA_STORAGE_FOLDER}/mysql:/var/lib/mysql
 
   zookeeper:
-    image: confluentinc/cp-zookeeper:5.2.1
+    image: confluentinc/cp-zookeeper:5.4.0
```

Comment:
Can you also bump up image versions for Kafka folder as well? `docker/kafka/docker-compose.yml`